src/leiningen/test_repeat.clj

```diff
@@ -0,0 +1,26 @@
+;; Copyright (c) 2016 BigML, Inc
+;; All rights reserved.
+
+;; Author: Chee Sing Lee <lee@bigml.com>
+;; Start date: Mon Aug 08, 2016 15:22
+
+(ns leiningen.test-repeat
+  "Repeat a test multiple times"
+  (:require [leiningen.core.main :as lein]
+            [leiningen.core.eval :as lein-eval]
+            [leiningen.core.project :as lein-project]))
+
+(defn test-repeat
+  "Repeat a test multiple times"
+  [project n & tests]
```

Comment:
it'd be nice to have a "repeat until it fails", with `n` being an upper bound to the number of executions... is that something doable easily in this same pr?